### PR TITLE
Bug 1184026 - Part 1 of 1:  Integrated speech recognition models into the language pack builder tool

### PR DESCRIPTION
**Pull request by kdavis-mozilla**
See [Bug 1184026](https://bugzilla.mozilla.org/show_bug.cgi?id=1184026)

### CODE REVIEW

File: bin/lp-builder.js [hunk 1]
```diff
@@ -22,12 +22,13 @@ var config = {
   LP_VERSION: null,
   LP_APPS: null,
   LP_TASKS: ['copy', 'appmanifests', 'optimize'],
+  LP_GLOBAL_TASKS: ['copySpeechData'],
```
Comment (stasm) [inline bin/lp-builder.js:25]: Do we need global tasks in addition to regular ones?  It seems that `copySpeechData` ends up in `tasks` anyways.

Comment (kdavis-mozilla) [inline bin/lp-builder.js:25]: Each `LP_TASKS` element is executed once for each app/locale combination, see `LangpackBuilder.prototype.build`. However, each `LP_GLOBAL_TASKS` element is executed once per locale.

Comment (stasm) [inline bin/lp-builder.js:25]: OK, that makes sense.  Can we rename `LP_TASKS` to `LP_APP_TASKS` and `LP_GLOBAL_TASKS` to `LP_LOCALE_TASKS`?  I can also do this in a follow-up.

Comment (kdavis-mozilla) [inline bin/lp-builder.js:25]: I thought about doing that, but I didn't want to "step on anyone's toes".

**Comment by stasm** [comment]
@kdavis-mozilla Could you resubmit this to https://github.com/l20n/langpack-builder please?

Do you want to rename `LP_TASKS` in this pull request, or should we do it in another one?

**Comment by kdavis-mozilla** [comment]
I submitted a pull request to [l20n/langpack-builder](https://github.com/l20n/langpack-builder). It doesn't include the `LP_TASKS` change, sorry.

**Comment by stasm** [comment]
That's perfect, I'll be happy to make the change myself.

**Pull request merged**